hack/build-test-images.sh

```diff
@@ -27,3 +27,5 @@ buildimage sti_test/sti-fake-user test/integration/images/sti-fake-user
 buildimage sti_test/sti-fake-scripts test/integration/images/sti-fake-scripts
 buildimage sti_test/sti-fake-scripts-no-save-artifacts test/integration/images/sti-fake-scripts-no-save-artifacts
 buildimage sti_test/sti-fake-no-tar test/integration/images/sti-fake-no-tar
+buildimage sti_test/sti-fake-onbuild-source test/integration/images/sti-fake-onbuild-source
+buildimage sti_test/sti-fake-onbuild test/integration/images/sti-fake-onbuild
```

Comment:
my thought was since ONBUILD doesn't take effect until another image uses it as a FROM we would need to build one as a source first and then we can test to make sure the ONBUILD was triggered with this image..